lib/output.js

```diff
@@ -53,17 +55,18 @@ function processOutputs(outputs) {
   return outputObj;
 }
 
-function createOutput(outputs, outFile, baseURL, opts) {
+function createOutput(outputs, outFile, outputPath, opts) {
   // process output
   var saucy = require('./sourcemaps');
   var sourceMap;
+  var onWindows =  process.platform.match(/^win/);
+  var outputURL = (onWindows ? 'file://' : 'file:/') + outputPath;
```

Comment:
To be super difficult, we're still in a world of `file://` for now... That is, one less slash!